### PR TITLE
Add demo gif

### PR DESCRIPTION
**Pull request by m0ham3dx**
This pull request , add a CLI demo of your tool. which covers 

1. Installation of the tool 
2. Usage of the tool 

I am also making this PR request via the github ui , if there are some errors do let me know if it requires corrections

### CODE REVIEW

**Comment by m0ham3dx** [comment]
There is gitpod.yaml file , please remove that , I shouldve have deleted that before making this PR

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: npm install
```
Comment (eliteWalrus4164) [inline .gitpod.yml]: Maybe we can consider adding .gitpod.yml to .gitignore?

File: README.md [hunk 2]
```diff
@@ -47,4 +47,9 @@ apiName is one of:
 
  contractAddress in hexadecimal format (i.e 0x1F98431c8aD98523631AE4a59f267346ea31F984)
 
- outDir specifies optionally where the sources will be downloaded relative to current directory
\ No newline at end of file
+ outDir specifies optionally where the sources will be downloaded relative to current director
```
Comment (eliteWalrus4164) [inline README.md]: Please fix the small typo introduced :)

**Review by eliteWalrus4164 — COMMENTED** [review]
Hello @m0ham3dx, thanks for the nice gif! There are a few minor details which would be nice to fix before merging the PR, otherwise looks good!

File: .gitpod.yml (hunk 1 above)
Comment (m0ham3dx) [reply .gitpod.yml]: File added to .gitignore , but then also thought that users of the package dont need to have this file, sine it is only relevant for users of [gitpo.io](https://gitpod.io/) which is an online IDE , to file has been deleted in the latest commit

File: README.md (hunk 2 above)
Comment (m0ham3dx) [reply README.md]: - Type fixed
- Also added new descriptive line L55 in readme.md

**Pull request merged**